.github/workflows/on-remove.yml

```diff
@@ -147,14 +147,22 @@ jobs:
         uses: ./.github/actions/setup-environment
         with:
           environment-name: Branch
-      - name: Contentful Teardown
-        id: contentful-teardown
+      - name: Contentful Environment Teardown
+        id: contentful-env-teardown
         shell: bash
         run: bash .github/scripts/contentful/teardown.sh
         env:
           CONTENTFUL_SPACE_ID: ${{ steps.setup.outputs.contentful-space-id }}
           CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: ${{ secrets.CONTENTFUL_MANAGEMENT_TOKEN }}
           CONTENTFUL_ENV_ID: ${{ steps.setup.outputs.crn-contentful-env-id }}
+      - name: Contentful Webhook Teardown
+        id: contentful-webhook-teardown
+        shell: bash
+        run: yarn workspace @asap-hub/contentful space:remove-webhook
+        env:
+          CONTENTFUL_SPACE_ID: ${{ steps.setup.outputs.contentful-space-id }}
+          CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: ${{ secrets.CONTENTFUL_MANAGEMENT_TOKEN }}
+          CONTENTFUL_ENVIRONMENT: ${{ steps.setup.outputs.crn-contentful-env-id }}
```

Comment:
It might not matter, but is there a chance that the environment teardown fails - if for example the environment was manually deleted to free space - that this leaves lingering webhooks behind? Should the two steps run in parallel?

Reply:
well actually if it fails to take down the contetnful environment then it shouldn't really proceed to deleting the webhook, should it? so imho that's the best setup, doing it sequentially, I mean.